unstructured/staging/base.py

```diff
@@ -39,6 +39,35 @@ def _get_metadata_table_fieldnames():
 ] + _get_metadata_table_fieldnames()
 
 
+def convert_to_text(elements: List[Element]) -> str:
+    """Converts a list of elements into clean, concatenated text."""
+    clean_concat_text: str = ""
+    for element in elements:
```

Comment:
Two questions: why do we need to run `to_dict()` rather than simply get the text from the element using `element.text`? And do we always want to concatenate using a newline character?

Reply:
Simplification to use element.text added now from other comments :)
On newline, I'd say part of the benefit of separating out the actual text conversion logic (even though its only one line) is that we can change maybe the joining character or any other text formatting we may want to do. For now, we do always want to concatenate with a single new line character, so it's not currently a parametrized value.